modules/formulize/admin/save/element_options_save.php

```diff
@@ -251,6 +251,31 @@
   }
 }
 
+// replace all existing blank radio button selections to the currently selected default
+if(isset($_POST['replaceblankswithdefault']) AND $_POST['replaceblankswithdefault']==1 AND $ele_type == "radio") {
+  $default_entry = (isset($_POST['changeuservalues']) AND $_POST['changeuservalues']==1) ? array_search(1, $newValues) : array_search(1, $element->getVar('ele_value'));

```

Comment:
This code runs when the page is saved, and the new default value has not been stored in the database yet, so it's using the previously saved value from the database, which might be wrong.